### DISCUSSION TITLE
Using the Fluent.Ribbon with Prism

### DISCUSSION THREAD

**Discussion by Yucca771** [fluentribbon/Fluent.Ribbon#1199] [Q&A] [answered]
When I try and use the Fluent.Ribbon with Prism it will not render and gives the following error: -

System.InvalidOperationException: 'Window must be the root of the tree. Cannot add Window as a child of Visual.'

Is there a working example anywhere or can anyone help?

**Comment by batzen** [comment, marked as answer]
I don't have any experience with Prism, but it looks like you are trying to add a `Window` (i guess a `RibbonWindow`) as the child element of another control.
Maybe your code to define a view uses `RibbonWindow` instead of `Ribbon`?
